Dockerfile

```diff
@@ -1,18 +1,25 @@
-FROM python:3.7-alpine
+FROM python:3.7.5-alpine
 
-RUN apk add --no-cache gcc musl-dev
+RUN mkdir -p /usr/pyhydroquebec
 
-WORKDIR /usr/src/app
+RUN mkdir -p /usr/pyhydroquebec/config
 
-COPY requirements.txt ./
-RUN pip install -r requirements.txt --force-reinstall --no-cache-dir
+WORKDIR /usr/pyhydroquebec
```

Comment:
Why do you change the working dir ?

Reply:
I put everything in the same root folder instead of having one root folder for the code and one root folder for the config.

Reply:
OK, but `/usr/SOMETHING` it's a weird folder. I would use something like `/usr/local/pyhydroquebec`.
In fact, we are copyng application's sources, that's why I put `/usr/src/app`, but I should put `/usr/local/src/pyhydroquebec`.

What do you think using `/usr/local/src/pyhydroquebec` ?